src/helix-ui/styles/vars.less

```
@@ -92,10 +92,11 @@
 @app-nav-width: 15rem;
 
 // ==== Z-INDEX =====
+@drawer-z-index: 50;
 @menu-z-index: 100;
 @search-assistance-z-index: 100;
-@tooltip-z-index: 1000;
-@popover-z-index: 1100;
+@popover-z-index: 1000;
```

Was this intended as a line switch? If that was the intention, the numbers need to be switched.
```suggestion
@popover-z-index: 1100;
```

---

This was intentional. 

If a tooltip and popover are in close proximity and would overlap, a tooltip (having transient interactivity) should always sit above a popover (having more static interactivity).